.travis.yml

```diff
@@ -0,0 +1,24 @@
+---
+branches:
+  - only:
+      - master
+
+language: python
+cache: pip
+
+python:
+  - "3.6"
+  - "nightly"
+
+# command to install dependencies
+install: "pip install -r requirements.txt pylint flake8 yamllint pydocstyle"
+
+# command to run tests
+script:
+  - flake8 jenkins_node_scanner.py --max-line-length 90 -v
+  - pylint jenkins_node_scanner.py --max-line-length 90
+  - pydocstyle jenkins_node_scanner.py .
+  - yamllint .travis.yml
+
+# use new travis-ci container-based infrastructure
+sudo: false
```

Comment:
@mst-ableton: that's the default (see https://docs.travis-ci.com/user/reference/overview/#Virtualization-environments), you can remove these two lines.